src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -348,7 +348,7 @@ bool emberAfOperationalCredentialsClusterAddOpCertCallback(chip::app::Command *
     AdminPairingInfo * admin = retrieveCurrentAdmin();
     VerifyOrExit(admin != nullptr, status = EMBER_ZCL_STATUS_FAILURE);
 
-    VerifyOrExit(admin->SetOperationalCert(OperationalCert) == CHIP_NO_ERROR, status = EMBER_ZCL_STATUS_FAILURE);
+    VerifyOrExit(admin->SetOperationalCertsFromCertArray(OperationalCert) == CHIP_NO_ERROR, status = EMBER_ZCL_STATUS_FAILURE);
```

Comment:
Suggest renaming OperationalCert argument to NOCArray. I will open a spec PR to renormalize this, now that OperationalCert argument was renamed NOC.

Reply:
See https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/3437 for some background